src/main/java/ca/corefacility/bioinformatics/irida/service/impl/AnalysisExecutionScheduledTaskImpl.java

```diff
@@ -52,35 +58,42 @@
 	private GalaxyJobErrorsService galaxyJobErrorsService;
 	private JobErrorRepository jobErrorRepository;
 	private final EmailController emailController;
+	private IridaFileStorageUtility iridaFileStorageUtility;
 	private AnalysisWorkspaceService analysisWorkspaceService;
-
+	private AnalysisSubmissionTempFileRepository analysisSubmissionTempFileRepository;
+	private StorageType storageType;
```

Comment:
I see we're passing the `StorageType` around a lot.  Since we're already passing the `iridaFileStorageUtility`, can we just make that a property of that object?  That'll remove a dependency and save wiring `StorageType` around all over the place.

Reply:
Updated storage utilities to have storageType as a property in eb2d3b4fd2ef1fc91ed99b94fb0574c6bc579ea9